app/build.gradle

```diff
@@ -62,42 +66,39 @@ ktfmt {
 }
 
 dependencies {
-    implementation 'androidx.core:core-ktx:1.9.0'
-    implementation 'androidx.appcompat:appcompat:1.5.1'
-    implementation 'com.google.android.material:material:1.6.1'
+    implementation 'androidx.core:core-ktx:1.10.0'
+    implementation 'androidx.appcompat:appcompat:1.6.1'
+    implementation 'com.google.android.material:material:1.8.0'
     implementation "androidx.compose.ui:ui:$compose_version"
     implementation "androidx.compose.material:material:$compose_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
     implementation "androidx.compose.material:material-icons-extended:$compose_version"
-    implementation 'androidx.lifecycle:lifecycle-runtime-compose:2.6.0-alpha02'
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
-    implementation 'androidx.activity:activity-compose:1.6.0-rc02'
-    implementation "androidx.navigation:navigation-compose:2.6.0-alpha01"
+    implementation 'androidx.lifecycle:lifecycle-runtime-compose:2.6.1'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.6.1'
+    implementation 'androidx.activity:activity-compose:1.8.0-alpha03'
+    implementation 'androidx.navigation:navigation-compose:2.6.0-alpha09'
```

Comment:
I usually refrain from using alpha versions in sample apps. Does this version bring a feature that we explicitly need?

Reply:
We were already using an alpha version before, that got updated (from `alpha06` to `alpha09` only), then I updated the `activity-compose` one to `alpha03` as well for compatibility. I believe we won't have any problems with it, and I'll start adding new thing to this app next week, including UI stuff, so I'm planning on doing some experiments.